_data/tools.yml

```diff
@@ -0,0 +1,25 @@
+
+- name: DigInG Icon Pack
+  desc: "Web icons made for commercial or personal use."
+  description: "Aute qui officia qui officia quis dolore in adipisicing laboris aliqua aute fugiat eu." 
+  url: https://github.com/diging/diging-icon-pack
+
+- name: JSTOR Importer
+  desc: "Import JSTOR data for corpus analysis and Elasticsearch."
+  description: "Aute qui officia qui officia quis dolore in adipisicing laboris aliqua aute fugiat eu." 
+  url: https://github.com/diging/jstor-import
+
+- name: Place Name Resolver
+  desc: "Retrieve Wikipedia titles and their geographic coordinates."
```

Comment:
Make that: "Match place names with Wikipedia titles and retrieve their geographic coordinates."